<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationWindow</class>
 <widget class="QDialog" name="InformationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Strategy</string>
    </attribute>
    <widget class="QLabel" name="imgLeonalbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgPrestolbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgGandalflbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="channelGandalflbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityGandalflbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="channelLeonalbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityLeonalbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="channelPrestolbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityPrestolbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pidlbl">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityMaxGandalflbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed Máx:</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityMaxLeonalbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed Máx:</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityMaxPrestolbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed Máx:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pidbtn">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>30</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QPushButton" name="velocityMaxGandalfbtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
     <property name="shortcut">
      <string>F5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="velocityMaxLeonabtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QPushButton" name="velocityMaxPrestobtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>340</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QLabel" name="strategylbl">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Strategy:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="strategycbx">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>130</y>
       <width>103</width>
       <height>36</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Text</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="strategybtn">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>130</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QLabel" name="gandalflbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gandalf</string>
     </property>
    </widget>
    <widget class="QLabel" name="leonalbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Leona</string>
     </property>
    </widget>
    <widget class="QLabel" name="prestolbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Presto</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Robots</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="velocityMaxGandalfspb">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="velocityMaxLeonaspb">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="velocityMaxPrestospb">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pidspb">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QLabel" name="currentStrategylbl">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>190</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current: </string>
     </property>
    </widget>
    <zorder>imgLeonalbl</zorder>
    <zorder>imgPrestolbl</zorder>
    <zorder>imgGandalflbl</zorder>
    <zorder>channelGandalflbl</zorder>
    <zorder>velocityGandalflbl</zorder>
    <zorder>channelLeonalbl</zorder>
    <zorder>velocityLeonalbl</zorder>
    <zorder>channelPrestolbl</zorder>
    <zorder>velocityPrestolbl</zorder>
    <zorder>pidlbl</zorder>
    <zorder>velocityMaxGandalflbl</zorder>
    <zorder>velocityMaxLeonalbl</zorder>
    <zorder>velocityMaxPrestolbl</zorder>
    <zorder>strategylbl</zorder>
    <zorder>strategycbx</zorder>
    <zorder>strategybtn</zorder>
    <zorder>gandalflbl</zorder>
    <zorder>leonalbl</zorder>
    <zorder>prestolbl</zorder>
    <zorder>label_39</zorder>
    <zorder>velocityMaxGandalfspb</zorder>
    <zorder>velocityMaxLeonaspb</zorder>
    <zorder>velocityMaxPrestospb</zorder>
    <zorder>pidspb</zorder>
    <zorder>currentStrategylbl</zorder>
    <zorder>velocityMaxGandalfbtn</zorder>
    <zorder>velocityMaxLeonabtn</zorder>
    <zorder>pidbtn</zorder>
    <zorder>velocityMaxPrestobtn</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Vision</string>
    </attribute>
    <widget class="QLabel" name="velocityPrestolbl_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgLeonalbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityGandalflbl_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityLeonalbl_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgGandalflbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="positionGandalflbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Posição:</string>
     </property>
    </widget>
    <widget class="QLabel" name="positionPrestolbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Posição:</string>
     </property>
    </widget>
    <widget class="QLabel" name="positionLeonalbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Posição:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgPrestolbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="velocityBalllbl">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgBalllbl">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="positionBalllbl">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Posição:</string>
     </property>
    </widget>
    <widget class="QLabel" name="leonalbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Leona</string>
     </property>
    </widget>
    <widget class="QLabel" name="prestolbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Presto</string>
     </property>
    </widget>
    <widget class="QLabel" name="gandalflbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Gandalf</string>
     </property>
    </widget>
    <widget class="QLabel" name="balllbl">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bola</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startFilterbtn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QLabel" name="chopFrequencylbl">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>150</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Chop Frequency:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="filterspb">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>190</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="valueFilterchangebtn">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>190</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
     <property name="shortcut">
      <string>F5</string>
     </property>
    </widget>
    <widget class="QLabel" name="frequencyFilterlbl">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>190</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
